src/Component/src/Metadata/Resource/Factory/AttributesResourceNameCollectionFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Metadata\Resource\Factory;
```

Comment:
I think it will be better like this
```suggestion
namespace Sylius\Resource\Metadata\Resource\NameCollection\Factory;
```

Reply:
But in this, case we should move The ResourceNameCollection too, and moved all the service related to MetataCollection in another namespace too.

Reply:
Discussed in slack; Since NameCollection is just a VO, then extensive namespacing is not needed and current one looks good enough.